src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -266,11 +269,13 @@ private static bool GetTls13Support()
                 // [ActiveIssue("https://github.com/dotnet/runtime/issues/1979")]
                 return false;
             }
```

Comment:
Does the above ActiveIssue related to iOS as well?

Reply:
I don't know. This will probably show up as we get more tests running.